src/Nethermind/Nethermind.Core/Crypto/KeccakCache.cs

```diff
@@ -21,6 +21,7 @@ namespace Nethermind.Core.Crypto;
 /// </summary>
 public static unsafe class KeccakCache
 {
+    private static readonly uint s_instanceRandom = (uint)System.Security.Cryptography.RandomNumberGenerator.GetInt32(int.MinValue, int.MaxValue);
```

Comment:
Ah yes, the initial seed so that nobody finds a way to attack all the nodes. 💪 @benaadams